account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -874,10 +874,23 @@ def _update_taxes(self, t_data):
                         real_account_id_xml_id = (
                             f"account.{self.company_id.id}_{account_id_xml_id}"
                         )
-                        if self.env.ref(
+                        account = self.env.ref(
                             real_account_id_xml_id, raise_if_not_found=False
-                        ):
-                            new_repartition_line_ids.append(line)
+                        )
+                        if account:
+                            new_cmd = (
+                                line[0],
+                                line[1],
+                                {**line[2], "account_id": account.id},
+                            )
+                        else:
+                            line[2].pop("account_id")
+                            new_cmd = (
+                                line[0],
+                                line[1],
+                                {**line[2]},
+                            )
```

Comment:
Let's optimize the code:

```suggestion
                        if account:
                            line[2]["account_id"] = account.id
                        else:
                            line[2].pop("account_id")
                        new_cmd = (line[0], line[1], line[2])
```

Reply:
Anyway, I'm thinking that this can be because the account is not yet created, and this has to be deferred somehow until the account is created.